data/presets/amenity/public_bookcase.json

```diff
@@ -3,23 +3,28 @@
     "fields": [
         "name",
         "public_bookcase/type",
+        "books",
         "operator",
         "capacity",
-        "lit"
```

Comment:
The `lit` tag seems to be a relatively popular combination (see taginfo: https://taginfo.openstreetmap.org/tags/amenity=public_bookcase#combinations). Why hide it?

Reply:
For clarity, it is moved to moreFields. (it is not hidden if people can easily discover it)
I think it is overall more of an exception they are lit, although relatively popular now.
Similarly, I think it is exception people will use a bookcase at night (except in areas with longer nights).
Otherwise I also don't think it is significant property of object itself, like for advertising.